test/parallel/test-worker-fs-stat-watcher.js

```diff
@@ -0,0 +1,18 @@
+// Flags: --experimental-worker
+'use strict';
+const common = require('../common');
+const { Worker, isMainThread, parentPort } = require('worker_threads');
```

Comment:
It looks like `isMainThread` isn't used.

Reply:
Thanks, you’re right!